R/ruby/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ruby"
+version = v"2.6.6"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource(
+        "https://cache.ruby-lang.org/pub/ruby/2.6/ruby-2.6.6.tar.gz",
+        "364b143def360bac1b74eb56ed60b1a0dca6439b00157ae11ff77d5cd2e92291",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ruby-2.6.6/
+apk add ruby-full
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-baseruby=/usr/bin/ruby --enable-shared
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter(
+    i -> (i isa Linux && i.libc === :glibc) || i isa FreeBSD,
```

Comment:
Perhaps add a brief comment that gives a hint why Linux with Musl libc and macOS are skipped? That will help "future generations" who might want to help with this :-)

Reply:
There's really not a lot of hints to give here. I had a quick look at the error messages they through and thought I wouldn't bother for now. :grin:

Reply:
FreeBSD probably just needs a patch or two from the ports tree, e.g. https://svnweb.freebsd.org/ports/head/lang/ruby26/files/patch-configure.ac?view=markup

Reply:
Do you mean to fix the warnings? Otherwise, FreeBSD is already building.

Reply:
Sorry, deleted my comment, I misread initially.